applications/FreeSurfaceApplication/custom_python/add_custom_io_to_python.cpp

```diff
@@ -134,5 +134,7 @@ void  AddCustomIOToPython(pybind11::module& pymodule)
 }
 }  // namespace Python.
 
+template class GidIO<>;
```

Comment:
I think he problem comes from the `GidIO<EdgebasedGidGaussPointsContainer,EdgebasedGidMeshContainer>`.

I don't have my win machine here right now, I will test it tomorrow

Reply:
Hi @roigcarlo, I made more changes, maybe this are more in the way of solving the problem, in linux everything still works properly.